src/components/navigation/nav-tab-panel.tsx

```diff
@@ -103,7 +103,7 @@ export class NavTabPanel extends BaseComponent<IProps> {
       case ENavTab.kStudentWork:
         return <StudentGroupView/>;
       case ENavTab.kSortWork:
-        return <SortWorkView />;
+        return <SortWorkView tabSpec={tabSpec}/>;
```

Comment:
This is going to be used in the next ticket (opening documents)

Reply:
I am now thinking that it would make more sense to leave this out of this PR.  It makes tracking the changes and functionality easier, and will make reviewing more clear.